src/library_html5.js

```diff
@@ -2652,7 +2652,7 @@ var LibraryJSEvents = {
   },
 
   emscripten_webgl_destroy_context_calling_thread: function(contextHandle) {
-    if (GL.currentContext == contextHandle) GL.currentContext = 0;
+    if (GL.currentContext == contextHandle) GL.currentContext = null;
```

Comment:
This actually regresses code size in interesting code, but getting to closure warnings clean is more urgent than micro-optimizing size. I might look at reissuing these kinds of micro-opts once at a later time.

Reply:
I'd hope this wouldn't regress size, won't closure emit a single `var x = null;` at the top, then do `=x` here (so still a single character)?

Reply:
Closure never outlines anything, functions or variables. So it will never do things like `var x = undefined;` and reference `x` instead of undefined. Likewise for null or any other variable. (not sure if it does common subexpression elimination either, likely not)

Reply:
It doesn't do general outlining, yeah, but I've seen it specifically define `null` and `undefined` at the very top, then reuse those. However, testing again now, it doesn't... so that was years ago and has changed it seems.